connectors/kinesis-source/3.0/kinesis-source.md

```diff
@@ -0,0 +1,177 @@
+---
+description: The Kinesis source connector pulls data from Amazon Kinesis and persists data into Pulsar
+author: ["ASF"]
+contributors: ["ASF"]
+language: Java
+document:
+license: Apache License 2.0
+tags: ["Pulsar IO", "Kinesis", "Source"]
+alias: Kinesis Source
+features: ["Use Kinesis source connector to sync data from Pulsar"]
+license_link: "https://www.apache.org/licenses/LICENSE-2.0"
+icon: /images/connectors/kinesis.svg
+support: StreamNative
+support_link: https://streamnative.io
+support_img: "/images/streamnative.png"
+owner_name: ""
+owner_img: ""
+dockerfile: https://hub.docker.com/r/streamnative/pulsar-io-kinesis
+id: "kinesis-source"
+source: "https://github.com/streamnative/pulsar/tree/v3.0.0.1/pulsar-io/kinesis"
+download: "https://github.com/streamnative/pulsar/releases/download/v3.0.0.1/pulsar-io-kinesis-3.0.0.1.nar"
+sn_available: true
+---
+
+The Kinesis source connector pulls data from Amazon Kinesis and persists data into Pulsar. For more information about connectors, see [Connector Overview](https://docs.streamnative.io/docs/connector-overview).
+
+![](/images/connectors/kinesis-source.png)
+
+This connector uses the [Kinesis Consumer Library](https://github.com/awslabs/amazon-kinesis-client) (KCL) to consume messages. The KCL uses [DynamoDB](https://docs.aws.amazon.com/streams/latest/dev/shared-throughput-kcl-consumers.html) to track checkpoints for consumers,
+and uses [CloudWatch](https://docs.aws.amazon.com/streams/latest/dev/monitoring-with-cloudwatch.html) to track metrics for consumers.
+
+This document introduces how to get started with creating an AWS Kinesis source connector and get it up and running.
+
+> #### Note
```

Comment:
Does it make sense to align the style/format for note? Now we have three different ones on a single page. See [2](https://github.com/streamnative/pulsar-hub/pull/296/files#diff-9888b895b61b8758a66b48d2db388cf2ac4ad456216bf8ebb12d06b14e801f3fR70) and [3](https://github.com/streamnative/pulsar-hub/pull/296/files#diff-9888b895b61b8758a66b48d2db388cf2ac4ad456216bf8ebb12d06b14e801f3fR101).

Reply:
Yes, Thanks for your feedback.